src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15443.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue15443 : _IssuesUITest
+	{
+		public override string Issue => "Programmatic position set should work as expected";
+
+		public Issue15443(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.CarouselView)]
+		public void CarouselShouldWorkProperOnBinding()
```

Comment:
Can update the sample to include a Label with a binding to the CarouselView position and verify it here in the test?

Reply:
Hi @jsuarezruiz , I have updated the sample to include a Label that's bound to the `CarouselView` Position property. The label updates dynamically as the carousel position changes. I have also added verification in the test to ensure that the label reflects the correct position during navigation.